src/index.ts

```diff
@@ -217,6 +229,15 @@ export class ProjectConfig {
     if (this.allFragments.length === 0) {
       this.allFragments.push(this.entrypoint);
     }
+
+    /**
+     * builds
+     */
+    if (Array.isArray(options.build)) {
+      this.builds = options.build;
+    } else if (options.build) {
+      this.builds = [options.build];
```

Comment:
if `options.build` is `undefined`, the `if (options.build)` conditional wouldn't pass and this path wouldn't be hit.